target/classes/templates/Fundraisers.html

```diff
@@ -29,34 +28,24 @@ <h1>Rewards</h1>
         <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRX7YN7ydBMKy7GM52M45M4HUV2Mn_J567UNQ&usqp=CAU" alt="Fundraiser Placeholder" width="300" height="200">
 
         <div id ="reward3-screen">
-            <h2>Earnable Tier 3 Rewards</h2>
-            <table>
-                <tr>
-                    <td>
-                        Tier 3 Redeemable Rewards
-                        <ul>
-                            <li>Book</li>
-                            <li>Cookies</li>
-                            <li>Wireless Mouse</li>
-                            <li>Printer</li>
-                            <li>Phone Accessories</li>
-                        </ul>
-                    </td>
-                    <h2>Tier 3 Information</h2>
-                    <form id="tier3-form">
-                        <label for="fun3Name">Fundraiser name: </label>
-                        <input type="text" id="fun3Name" name="fun3Name"><br><br>
-                        <p>Fundraiser Name: Help the Children</p>
-                        <label for="fun3Desc">Fundraiser Description: </label>
-                        <input type="text" id="fun3Desc" name="fun3Desc"><br><br>
-                        <p>Fundraiser Description: Help the Children is a fundraiser that will be working</p>
-                        <p>toward helping children in third-world countries.</p>
-                        <label for="amtRequired3">Amount Required for reward: </label>
-                        <input type="text" id="amtRequired3" name="amtRequired3"><br><br>
-                        <input type="submit" value="Submit">
-                    </form>
-                </tr>
-            </table>
+            <h2>Example Rewards</h2>
+                <ul>
+                    <li>Book</li>
+                    <li>Cookies</li>
+                    <li>Wireless Mouse</li>
+                    <li>Printer</li>
+                    <li>Phone Accessories</li>
+                </ul>
+                <h2>Tier Creation Form</h2>
+                <form id="tier3-form">
+                    <label for="tierName">Tier name: </label>
+                    <input type="text" id="tierName" name="fun3Name"><br><br>
+                    <label for="tierDesc">Tier Description: </label>
+                    <input type="text" id="tierDesc" name="fun3Desc"><br><br>
+                    <label for="amtRequired3">Amount Required for reward: </label>
+                    <input type="text" id="amtRequired3" name="amtRequired3"><br><br>
+                    <input type="submit" value="Submit">
+                </form>
```

Comment:
Good.  Use divs for formatting, not tables.

Tables should only be used for tabular data.